src/position.rs

```diff
@@ -346,17 +336,24 @@ impl From<shogi_core::PartialPosition> for PartialPosition {
 
 #[derive(Debug, Clone)]
 struct State {
+    /// Zobrist hashes for (board ^ side, hand)
+    /// TODO: It seems it's always used by XOR-ing two keys. Can we only store a single XOR-ed key?
     keys: (Key, Key),
+    /// Captured piece in the last move
```

Comment:
nit: choice of preposition, word order
```suggestion
    /// Piece captured on the last move
```

Reply:
Thanks for checking and correcting grammar issues!